jobs/build/ocp4/build.groovy

```diff
@@ -366,6 +366,17 @@ def stageBuildImages() {
 
         buildlib.doozer "${doozerOpts} images:streams mirror"
     }
+    // check new embargoed build
+    imagelist = buildPlan.imagesIncluded.replaceAll(',', ' ')
+    report = buildlib.doozer("${doozerOpts} -q detect-embargo images --version ${version.stream} ${imagelist}", [capture: true]).trim()
+    if (report.returnStatus != 2) {
```

Comment:
don't trust the exact return status; it doesn't always seem to match what bash returns. really all you can safely check here is whether it's zero or not.